quodlibet/quodlibet/qltk/__init__.py

```diff
@@ -19,6 +19,41 @@
 from senf import fsn2bytes, bytes2fsn
 
 from quodlibet.util import gdecode, print_d, print_w
+from quodlibet.compat import urlparse
+
+
+def show_uri(label, uri):
+    """Shows a uri. The uri can be anything handled by GIO or a quodlibet
+    specific one.
+
+    Currently handled quodlibet uris:
+        - ql:plugins:<plugin id>
+
+    Args:
+        label (str)
+        uri (str) the uri to show
+    Returns:
+        True on success, False on error
+    """
+
+    parsed = urlparse(uri)
+    if parsed.scheme == "quodlibet":
+        if parsed.netloc != "":
+            print_w("Unknown QuodLibet URL format (%s)" % uri)
+            return False
+        else:
+            return __show_quodlibet_uri(parsed)
+    else:
+        from quodlibet import app
+        return Gtk.show_uri_on_window(app.window, uri, 0)
```

Comment:
I have to test it again. IIRC, if I return False, it just writes an error in the console.
(Hopefully, Gtk.show_uri() is 3.14 and we just moved to that version ! :))